_health-care/_js/components/personal-information/Gender.jsx

```diff
@@ -1,45 +1,42 @@
 import React from 'react';
 import _ from 'lodash';
 
+/**
+ * Select component for getting a person's gender.
+ *
+ * No validation is provided since base UI elements provide all
+ * common options as provided in reference PDF/JS form.
+ */
 class Gender extends React.Component {
   constructor() {
     super();
-    this.state = { hasError: false };
     this.handleChange = this.handleChange.bind(this);
   }
 
   componentWillMount() {
-    this.id = _.uniqueId();
+    this.selectId = _.uniqueId('gender-select-');
   }
 
+  // handleChange(domEvent) {
   handleChange() {
-    const gender = this.refs.gender.value;
-
-    if (this.validate(gender)) {
-      this.setState({ hasError: false });
-    } else {
-      this.setState({ hasError: true });
-    }
-
-    this.props.onUserInput(gender);
+    this.props.onUserInput(this.refs.gender.value);
```

Comment:
So I think what you can do here instead of using refs is:
```javascript
handleChange(domEvent) {
  this.props.onUserInput(domEvent.target.value);
}
```
Then on the `select` below you can remove the `ref`.